ft_prepare_sourcemodel.m

```diff
@@ -46,7 +46,6 @@
 % inversely warped from MNI coordinates to the individual subjects MRI.
 % It uses the following configuration options:
 %   cfg.mri             = structure with anatomical MRI model or filename, see FT_READ_MRI
-%   cfg.warpmni         = 'yes'
```

Comment:
the `cfg.warpmni` option is now deprecated, people should use cfg.method instead